scripts/kiwix-resources

```diff
@@ -53,23 +53,18 @@ def set_cacheid(resource_matchobj):
     cacheid = 'cacheid=' + get_resource_revision(resource)
     return path + '?' + cacheid + extra_query
 
-def preprocess_line(line):
-    if 'KIWIXCACHEID' in line:
-        line = re.sub(RESOURCE_WITH_CACHEID_URL_PATTERN, set_cacheid, line)
-        assert not 'KIWIXCACHEID' in line
-    return line
+def preprocess_text(s):
+    if 'KIWIXCACHEID' in s:
+        s = re.sub(RESOURCE_WITH_CACHEID_URL_PATTERN, set_cacheid, s)
```

Comment:
I also approached this change as a simplification. Returning a single value instead of a pair of values that must be consistent with each other is inline with that goal. A simple comparison of the source content and the result of transforming it makes `preprocess_text()` easier to enhance should such a need arise.

Reply:
Yes. And this part is not critical at all. Perfect is the enemy of good. Let's move on.